docs/development.md

```diff
@@ -6,7 +6,7 @@
 
 Before building from source, be sure to install:
 
-- [Go 1.19.0+](https://golang.org/dl/)
+- [Go 1.19.0+](https://go.dev/dl/)
```

Comment:
nit: do we need to bump the golang ver here?

Reply:
Good call. IIRC, at some point, we starting depending on 1.22.